clients/go/zms/client.go

```diff
@@ -8,15 +8,14 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	rdl "github.com/ardielle/ardielle-go/rdl"
```

Comment:
why this change of imports ordering?  Could we keep the goimports-compatible order of imports (first stdlib, then empty line, then third-party packages) - otherwise if someone has goimports integrated with the code editor and opens/saves the file, causes spurious diffs.

Also in terms of consistency, I though the goimports-compatible imports ordering is already a de facto coding standard for Athenz - see for example https://github.com/yahoo/athenz/blob/master/clients/go/zms/auth.go#L6=L11 and https://github.com/yahoo/athenz/blob/master/clients/go/zms/client.go#L7-L19 that separate stdlib and third-party imports.